akka-http-core/src/main/resources/reference.conf

```diff
@@ -135,8 +135,11 @@ akka.http {
     # doesn't have to be fiddled with in most applications.
     request-header-size-hint = 512
 
-    # The proxy configurations to be used for requests with the specified
-    # scheme.
+    # The proxy configurations to be used for requests with the specified scheme.
+    # !! WARNING !!
+    # Client proxying is currently not implemented. Refer to https://github.com/akka/akka/issues/16853 for status.
+    # Community help on implementing this feature would be very welcome.
```

Comment:
Oh I thought it's surfaced in Settings objects, then it would be confusing to remove the setting (and we could not remove from APIs), but actually it seems this is not even surfaced in the Settings object so we can remove it completely